### DISCUSSION TITLE
Problems with gh student verification

### DISCUSSION THREAD

**Discussion by kiramishima** [community/community#137647] [GitHub Education]
### Hi everyone,

Hi everyone.

I am student Data Science at Rosario Castellanos University, However we dont have institutional email. I got GH Students Pack 2 years ago, I try to reverify but the system dont recognize my School ID and my list of subjects isnt valid because, they dont include a date only the period :1 

Hola soy estudiante de ciencia de datos en la universidad rosario castellanos, sin embargo tengo un problema al revalidar el student pack, mi universidad no nos da correo institucional, pero antes con la credencial revalidaba y ahora ya no me deja. Intente con la hoja de reiscripcion y tampoco la acepta, ya que no tiene una fecha de inicio solo ponen el periodo, en este caso 2024-2 .

**Comment by davevad93** [comment]
Hi @kiramishima ,

If your student ID doesn't have a current date, you can provide alternative documentation such as your class schedule, transcript, and affiliation or enrollment verification letter, the main thing is that the documentation you submit shows a current date that proves your student status.

**Reply by kiramishima** [reply]
It has expedition date and expiration date, we receive one student ID during 4 years. I upload also my class schedule but it isnt valid too.

This is the class schedule that the university gave me every semester ( I removed sensible data). In Google, they accept this in pdf format and the photo of my student ID without problem.

![Sans titre](https://github.com/user-attachments/assets/8fecc66d-9dcf-4f1a-8d6d-345db570ccf7)

**Reply by davevad93** [reply]
I see @kiramishima , however the problem with your class schedule is that it doesn't show a current date in a `DD/MM/YYYY` or `MM/DD/YYYY` format, considering that the application process is most likely handled by a bot I think it automatically rejects applications without a date in that format; I say this because last year when I applied I had the same problem and finally solved it by uploading a screenshot of my latest exam results with a `DD/MM/YYYY` date. If possible, try to provide some kind of documentation that meets this requirement.
